### DISCUSSION TITLE
Bearer not prefixed

### DISCUSSION THREAD

**Discussion by benborra** [Daniel-Krzyczkowski/Cars-Island-On-Azure#14] [General]
Hi!

Thanks for the great tutorial.
Small error I ran into:

https://github.com/Daniel-Krzyczkowski/Cars-Island-On-Azure/blob/0972fe2c3bb90e978d269110e2815eb5788d91ad/src/web-api/CarsIsland.API/Core/DependencyInjection/SwaggerCollectionExtensions.cs#L24

Had to change the type to `SecuritySchemeType.Http` to get the Bearer prefix to be included.
Not sure if this is expected behaviour and I'm missing something.

**Comment by Daniel-Krzyczkowski** [comment]
Thank you for the comment. It is because there was an update in the library. Please try to update NuGet Swashbuckle.AspNetCore to version 5.6.3.
